aten/src/ATen/native/native_functions.yaml

```diff
@@ -5361,14 +5361,29 @@
     CompositeExplicitAutograd: slice_backward
   autogen: slice_backward.out
 
+# NB: This op exists to back the implementation of reverse view_funcs for various views (chunk,
+# slice.Tensor, split_with_sizes, et. al.). Currently, these are only used during fake-ification
+# of PT2 graph input subclass instances that are views. This means:
+# * This op shouldn't really show up in eager mode (so e.g. XLA shouldn't have to implement it)
+# * This op shouldn't show up in a PT2 graph (so a PT2 backend shouldn't have to implement it)
```

Comment:
don't compositeexplicit ops show in the graph?

Reply:
in general yes they are, but we expect this particular op to only be invoked during fake-ification of an input subclass tensor instance, and op calls made during fake-ification aren't traced into the graph

Reply:
I had completely missed that sentence. Ooops